instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -167,13 +167,12 @@ def _determine_parent_context(
     """
     parent_context = None
 
-    if not disable_aws_context_propagation:
-        xray_env_var = os.environ.get(_X_AMZN_TRACE_ID)
+    xray_env_var = os.environ.get(_X_AMZN_TRACE_ID)
 
-        if xray_env_var:
-            parent_context = AwsXRayPropagator().extract(
-                {TRACE_HEADER_KEY: xray_env_var}
-            )
+    if xray_env_var:
+        parent_context = AwsXRayPropagator().extract(
+            {TRACE_HEADER_KEY: xray_env_var}
+        )
 
     if (
         parent_context
```

Comment:
You might need to do something here to ensure the event context extractor isn't invoked if there's already an active span context.

Reply:
Right ,fixed